python/startup/menu.py

```diff
@@ -0,0 +1,63 @@
+# Copyright (c) 2015 Shotgun Software Inc.
+# 
+# CONFIDENTIAL AND PROPRIETARY
+# 
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit 
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your 
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights 
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+import os
+
+def bootstrap_sgtk():
+    """
+    Bootstrapping routine for the Nuke mode of Nuke.
+    """
+    import nuke
+
+    _setup_sgtk(nuke.warning)
+
+    # Clean up temp env vars.
+    _clean_env()
+
+def _clean_env():
+    """
+    Cleans up SGTK related environment variables.
+    """
+    for var in ["TANK_ENGINE", "TANK_CONTEXT", "TANK_FILE_TO_OPEN"]:
+        if var in os.environ:
+            del os.environ[var]
+
+def _setup_sgtk(output_handle):
```

Comment:
@manneohrstrom , this looks like something that should be standardized in Toolkit